<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>f_SupprimerProfil</class>
 <widget class="QDialog" name="f_SupprimerProfil">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>215</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="Lb_Profil">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Profil Ã  supprimer :</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Cb_Bx_Profil">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>150</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_MDPAdmin">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe admin :</string>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_MDPAdmin">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="Bt_Bx_Valider">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>155</width>
     <height>25</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="Lb_MdP">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>115</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Mot de passe du profil :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_MdP">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>115</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
